GoogleForCreators/web-stories-wp
# Restyle the Save template button

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
N/A 


## Testing Instructions
1. Enable the custom page templates experiment.
2. Add an element to the page for it not to be empty.
3. Open the Page layouts panel in the library
4. Verify the button for Saving templates looks like [in Figma](https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7533%3A213534)
5. Click the button to save the page as a template.
6. Verify the view switches to the saved templates and the page appears there with blue focus outline appearing for 4 seconds.


### QA
See the steps above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #7219
